about.html

```diff
@@ -0,0 +1,87 @@
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="main.css">
+  <title>Capstone</title>
+</head>
+<body>
+  <header class="container=fluid">
+    <nav class="navbar nav1 navbar-expand-lg fixed-top nav_container d-inline-flex justify-content-end pb-0 pt-0"
+      role="navigation">
+      <ul class="">
+        <li><a href="#" class="nav-link1"><img src="assets/images/104498_facebook_icon.svg" alt="facebook icon"
+              class="nav-icons"></a></li>
+        <li><a href="#" class="nav-link1"><img src="assets/images/104501_twitter_icon.svg" alt="twitter icon"
+              class="nav-icons"></a></li>
+        <li><a href="#" class="nav-link1">English</a></li>
+        <li><a href="#" class="nav-link1">My Page</a></li>
+        <li><a href="#" class="nav-link1">Logouts</a></li>
+      </ul>
+    </nav>
+    <a class="menu justify-content-start"><img src="assets/images/menu_button.svg" alt="menu button"></a>
+
+    <nav class="navbar nav2 navbar-expand-lg fixed-top nav_container d-inline-flex justify-content-end"
+      role="navigation">
+      <ul class="">
+        <li id="small-logo"><a href="index.html"><img class="logo-small" src="assets/images/BTR-Logo.png" alt="btr logo"></a></li>
+        <li><a href="about.html" class="nav-link2">About</a></li>
+        <li><a href="#program" class="nav-link2">Program</a></li>
+        <li><a href="#" class="nav-link2">Join</a></li>
+        <li><a href="#" class="nav-link2">Sponsors</a></li>
+        <li><a href="index.html" class="nav-link2 logo">BTR Concert</a></li>
+      </ul>
+    </nav>
+    <div class="menu_opt">
+      <input type="button" name="close" value="X" class="close_menu" id="exit">
+      <ul class="menu_list">
+        <li class="menu_items"><a href="index.html">BTR Concert</a></li>
+        <li class="menu_items"><a href="about.html">About</a></li>
+        <li class="menu_items"><a href="#program">Program</a></li>
+        <li class="menu_items"><a href="#">Join</a></li>
+        <li class="menu_items"><a href="#">Sponsors</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <section class="about" id="about-page">
+    <div class="page-about">
+      <div class="overlay">
+        <div class="home-text about-text">
+          <h3>"It's about time to connect with your roots."</h3><br />
+          <h1>BACK TO ROOTS<br /> GLOBAL CONCERT 2012</h1>
+        </div>
+        <div class="home-p">
+          <p>A joyful celebration believing in the value of connecting with one's roots through musical entertainment.
+            Become the part of this year's concert by buying your ticket online.</p>
+        </div>
+        <div class="about-p2">
+          <p>Please contact us per Email for any further questions about the BTR Global Concert 2012!</p>
+          <p>info@brtgc.org</p>
+        </div>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] Ass the previous code reviewer said: **Please I repeat: You have to create About and Home separately in Both versions: Desktop and Mobile**
- [ ] Please 🙏🏽 the pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

**Desktop version**

|**Cindy's design** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192122814-542cab9d-78a7-4779-96b9-1d4084b47d6c.png) | 
![image](https://user-images.githubusercontent.com/58642949/192122831-ba20602d-c4e6-425d-aadd-f8c7a93a8dc1.png) |

**Mobile version**

| Cindy's design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192122844-53d7a4e1-dfab-440d-aa91-11e9f0bad3dd.png) | 
![image](https://user-images.githubusercontent.com/58642949/192122853-1a55ebc7-72ad-4095-ae00-67b96b105ea0.png) |